gnuradio-runtime/include/gnuradio/rpcbufferedget.h

```diff
@@ -34,7 +34,7 @@ class rpcbufferedget
         if (!d_data_needed)
             return;
         {
-            boost::mutex::scoped_lock lock(d_buffer_lock);
+            std::unique_lock lock(d_buffer_lock);
```

Comment:
I'm curious why not std::scoped_lock here

Reply:
Oh, I suppose this should be a `scoped_lock` since it's not needed for a condition variable.

Reply:
Fixed.